README.md

```diff
@@ -19,7 +19,7 @@ npm i -g yarn
 yarn install
 ```
 
-- Furthermore, to commit your changes use `yarn commit` instead of `git commit` for better commit experiance.
+- Furthermore, to commit your changes use `yarn commit` instead of `git commit` for better commit experience.
```

Comment:
Further more, to commit your changes use `yarn commit` instead of `git commit` for better commit experience.

Reply:
ok

Reply:
Furthermore is written together in this case, that is a complete word, that is correct

Reply:
Oh Yes